src/Controller/CocktailController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\CocktailFormType;
+use App\Entity\CocktailName;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+
+class CocktailController extends AbstractController
+{
+    #[Route('/home', name: 'home')]
```

Comment:
So this works only by going https://127.0.0.1/home? Without /home it would throw 404? (e.g. / or without any param)

Reply:
also, change "name" as later you will forget and will need to change it in multiple places if it gets used

Reply:
fixed